@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -57,82 +57,159 @@ export interface BaseTechniqueParams {
     fadeFar?: number;
 }
 
+export enum TextureCoordinateType {
+    /**
+     * Texture coordinates are in tile space.
+     * SW of the tile will have (0,0) and NE will have (1,1).
+     */
+    TileSpace = "tile-space",
+    /**
+     * Texture coordinates are in equirectangular space.
+     * (u, v) = (longitude / 360 - 180, latitude / 180 + 90).
+     */
+    EquirectangularSpace = "equirectangular-space"
+}
+
 /**
  * Standard technique paraneters.
  */
-export interface BaseStandardTechniqueParams extends BaseTechniqueParams {
+export interface StandardTechniqueParams extends BaseTechniqueParams {
```

Comment:
Agree, it is doing multiple things as stated in the commit message. I just wanted to avoid having inconsistent way of defining texture coordinates between "standard" and "extruded-polyline" technique.

Reply:
Sorry, I missed the extra parts of the commit message, just looked at the title, will be more careful in future. Ok.